modules/ROOT/pages/enable-https-mule3-proxies.adoc

```diff
@@ -9,7 +9,7 @@ Enable HTTPS in your Mule 3.x API proxies to enhance the privacy and protection
 To enable your proxy application to support HTTPS requests:
 
 . From API Manager, xref:download-proxy-task.adoc[download] and xref:proxy-latest-concept.adoc[deploy] an API proxy.
-. From Anypoint Studio, add a xref:4.3@mule-runtime::tls-configuration#use-anypoint-studio-to-configure-tls.adoc[keystore (SSL certificate)] to your API proxy (API gateway).
+. From Anypoint Studio, add a xref:4.3@mule-runtime::tls-configuration.adoc#use-anypoint-studio-to-configure-tls[keystore (SSL certificate)] to your API proxy (API gateway).
```

Comment:
Hmm, there is no 3.9 version of that file. It's about adding the certificate to Studio, which would mean a Studio 6 version. I don't see Studio docs on this topic.

For now, I'd remove the version from the xref so that it goes to the latest version of the Mule docs and probably double-check with Priya's team that they are ok with this.

Reply:
see commit suggestion for this below